src/SFML/Graphics/GLExtensions.hpp

```diff
@@ -41,28 +41,43 @@
 // All functionality beyond that is optional
 // and has to be checked for prior to use
 
+namespace sf::priv
+{
+// Stand-ins for desktop OpenGL feature-check flags
+// NOLINTBEGIN(readability-identifier-naming)
+inline int SF_GL_OES_multitexture         = 1;
+inline int SF_GL_OES_vertex_buffer_object = 1;
+// NOLINTEND(readability-identifier-naming)
+} // namespace sf::priv
+
+// Core since 1.0
+#define GLEXT_multitexture          sf::priv::SF_GL_OES_multitexture
```

Comment:
```suggestion
#define GLEXT_multitexture          ::sf::priv::SF_GL_OES_multitexture
```

I think that full qualification in macros is the clearest choice.